js/runner.go

```diff
@@ -861,3 +861,11 @@ func (s *scriptException) Hint() string {
 func (s *scriptException) ExitCode() exitcodes.ExitCode {
 	return exitcodes.ScriptException
 }
+
+func cloneRunTags(runTags map[string]string) map[string]string {
```

Comment:
this has nothing runTags specific, it might be better to just call it `copyStringMap(source map[string]string) map[string]string`?